test/test_hpyerr.py

```diff
@@ -402,64 +402,78 @@ def test_errorval_returned_by_api_functions_int(self):
         assert mod.f([100, 200, 300]) == 3
         assert mod.f(None) == -42
 
-    def test_HPyErr_NewException(self):
+    @pytest_collecting.mark.parametrize("doc", [None, b"mytest"])
+    @pytest_collecting.mark.parametrize('base', [True, False])
+    @pytest_collecting.mark.parametrize("dict_arg", [True, False])
+    def test_HPyErr_NewException(self, doc, base, dict_arg):
```

Comment:
why `True/False` for base and dict? I think it's simpler to just pass `None/RuntimeError` and `None/{...}`?

Reply:
Yes, I think it's easier and much cleaner. The other approach was due to the fact that I didn't feel very familiar with HPy.
Not that I do now, but after having written some stuff, it starts to feel better.

Reply:
yes sure, that's normal. I'm glad that you are getting used to HPy :)